browser/ui/webui/brave_rewards_ui.cc

```diff
@@ -143,6 +147,10 @@ class RewardsDOMHandler : public WebUIMessageHandler,
       brave_rewards::RewardsService* rewards_service,
       brave_rewards::ContentSiteList list) override;
 
+  void OnRemovePendingContribution(
```

Comment:
we shouldn't be using these for webui. The api methods should have callbacks that return the result. See tabs_api.cc for examples

Reply:
no